src/main/java/seedu/address/logic/commands/UnpaidCommand.java

```diff
@@ -25,9 +23,10 @@ public class UnpaidCommand extends Command {
     public static final String COMMAND_WORD = "unpaid";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Marks the person identified by the index number to have not made payment.\n"
+            + ": Marks the person identified by the index number to have "
+            + "not made payment and updates the policy renewal frequency to zero.\n"
             + "Parameters: INDEX (must be a positive integer)\n"
-            + "Example: " + COMMAND_WORD + " 1";
+            + "Example: " + COMMAND_WORD + " 1 ";
```

Comment:
Can check if the additional space here is intentional, and if so, why?

Reply:
The space is not intentional I'll get rid of it